crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -2153,6 +2226,18 @@ async fn test_custom_property_check_unpublished_dependencies() {
     expected.assert_eq(&error)
 }
 
+pub fn build_test_package(test_dir: &str, with_unpublished_deps: bool) -> Vec<Vec<u8>> {
+    let build_config = BuildConfig::new_for_testing();
+    let mut path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    path.push("src");
+    path.push("unit_tests");
+    path.push("data");
+    path.push(test_dir);
+    sui_framework::build_move_package(&path, build_config)
```

Comment:
Heads up @awelc / @rvantonder / @tzakian -- this will probably conflict with some of the package upgrades work.